pages/search.js

```diff
@@ -1,5 +1,6 @@
 import { useEffect, useState } from "react";
 import { useRouter } from "next/router";
+import { pick } from "lodash";
```

Comment:
Lodash is not a direct dependency & there is probably a way to do the same thing without including it.

Reply:
Good spot, yes we don't want to include `lodash`